ext/index/main.php

```diff
@@ -250,7 +250,15 @@ public function onSearchTermParse(SearchTermParseEvent $event)
             $seed = $matches[1];
             Image::$order_sql = "RAND($seed)";
             $event->add_querylet(new Querylet("1=1")); //small hack to avoid metatag being treated as normal tag
-        }
+        } elseif (preg_match("/^order[=|:]fullrandom$/i", $event->term, $matches)) {
```

Comment:
"fullrandom" seems a weird name to me, like it implies that it's fully random? For something which is random based on the day, I'd call it "daily random" or something like that o.o

Reply:
Yeah daily random sounds good :)

Reply:
I suggest to change it to "dailyshuffle" because it makes more sense.
The order changes daily, so random doesn't describe it properly